src/components/DashboardSidebarButtons/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import Button from "react-bootstrap/Button";
+
+import styles from "./styles.module.css";
+
+export default class DashboardSidebarButtons extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      text: props.text,
```

Comment:
Is this property needed for this button? I don't see it used anywhere

Reply:
Oops. You're right